lib/search/stream-filter.js

```diff
@@ -0,0 +1,10 @@
+var ms = require('mississippi')
+
+module.exports = function (filter) {
```

Comment:
Does this need to be a standalone file, given that it's only used in one place and is a pretty simple direct invocation?